enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -139,18 +139,26 @@ https://github.com/operator-framework/api/blob/v0.17.3/pkg/operators/v1alpha1/su
 We plan to make many of these fields optional, the controller can fill in required Subscription
 fields based on specified defaults in the operator's PackageManifest. For example, this allows the
 default channel to possibly be different on different clusters, reflecting the default channel in
-each cluster's catalog. Note: if the policy is in `enforce` mode and the allowed CSVs
-are restricted as specified below, then the `installPlanApproval` field on the Subscription will
-always be Manual, regardless of the setting here.
+each cluster's catalog. It is not allowed to set `spec.subscription.installPlanApproval`; the policy
+will choose the correct value based on the rest of the policy (it will often need to be "Manual" so
+that upgrades can be controlled, but requiring the user to set that, or overriding when the user
+specified "Automatic" is probably confusing behavior).
 
 In "musthave" mode, the `spec.versions` list specifies what installed versions are considered
 Compliant when the policy is in `inform` mode, and which InstallPlans can be approved when in
 `enforce` mode. Conversely, in "mustnothave" mode, this list specifies which versions are considered
 NonCompliant. Only exact matches are considered. If the list is unset or empty, then any version on
-the cluster will be considered a match.
+the cluster will be considered a match. If `spec.subscription.startingCSV` is set, that version can
+be approved, even if it is not in this list.
+
+The `spec.upgradeApproval` field specifies whether the an enforced "musthave" policy will approve
+any upgrade InstallPlans for the operator. *It has no effect when the policy is in "mustnothave"*
+*mode*. This only affects InstallPlans for operators that are already installed on the cluster,
+which upgrade or replace the operator; initial InstallPlans for an operator can be approved
+regardless of this flag. If not set, the controller will behave as if it were set to `Automatic`.
```

Comment:
I keep going back and forth on the default, but I think `Automatic` is a safe default option since upgrades should be safe within a channel.

Reply:
Did you mean `Automatic` is a good default in the OperatorPolicy, if this new field is not set? Or did you mean that `Automatic` in the Subscription is a good default? I updated this section some more, maybe it is more clear what I mean now.